upi/aws/cloudformation/04_cluster_bootstrap.yaml

```diff
@@ -88,7 +88,7 @@ Metadata:
       PublicSubnet:
         default: "Public Subnet"
       RhcosAmi:
-        default: "RHEL CoreOS AMI ID"
+        default: "Red Hat Enterprise Linux CoreOS AMI ID"
```

Comment:
Do we have to sorry this out all the time? I thought `RHEL CoreOS` was acceptable...

Reply:
In technical content, you're allowed to say "RHCOS" after the first use. Otherwise, it's always "Red Hat Enterprise Linux CoreOS." "RHEL CoreOS" was briefly a suggested shortname, but it's now specifically forbidden in the product names list.

Reply:
If folks are reading this, I think we can safely assume they're beyond first-use and we can use RHCOS.  I'm also ok spelling it out, so either way.

Reply:
First use = first use in a particular document. I'm glad you're ok with spelling it out because it needs to be. :)